laravel/app/helpers.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App;
+
+class Helpers {
```

Comment:
Please follow the indenting style of the rest of the project, braces on newlines. 

(Auto-linting would be nice :slightly_smiling_face:)

Reply:
You got it!